gsd/gsd.c

```diff
@@ -822,12 +909,12 @@ int gsd_end_frame(struct gsd_handle* handle)
         if (handle->open_flags != GSD_OPEN_APPEND)
             data += handle->index_written_entries;
 
-        size_t bytes_written = pwrite(handle->fd,
-                                     data,
-                                     sizeof(struct gsd_index_entry)*entries_to_write,
-                                     write_pos);
+        size_t bytes_written = __pwrite_retry(handle->fd,
```

Comment:
The bytes written should be of type `ssize_t` here, shouldn't it?
Like everywhere else.

Reply:
you are correct